modules/bullet/generic_6dof_joint_bullet.cpp

```diff
@@ -261,3 +264,28 @@ bool Generic6DOFJointBullet::get_flag(Vector3::Axis p_axis, PhysicsServer::G6DOF
 	ERR_FAIL_INDEX_V(p_axis, 3, false);
 	return flags[p_axis][p_flag];
 }
+
+void Generic6DOFJointBullet::set_use_global_rotation(bool p_value) {
+	sixDOFConstraint->set_use_global_rotation(p_value);
+}
+
+bool Generic6DOFJointBullet::get_use_global_rotation() {
+	return sixDOFConstraint->get_use_global_rotation();
+}
+
+void Generic6DOFJointBullet::set_use_quaternion_rotation_equilibrium(bool p_enable) {
+	sixDOFConstraint->set_use_quaternion_rotation_equilibrium(p_enable);
+}
+
+bool Generic6DOFJointBullet::get_use_quaternion_rotation_equilibrium() {
+	return sixDOFConstraint->get_use_quaternion_rotation_equilibrium();
+}
+void Generic6DOFJointBullet::set_quaternion_rotation_equilibrium(Quat p_value) {
```

Comment:
```suggestion
}

void Generic6DOFJointBullet::set_quaternion_rotation_equilibrium(Quat p_value) {
```

The ones that I pointed out are not the only ones, there are like ten more.